src/components/form-control/form-control.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+
+import { useFormControlStyles } from './form-control.styles';
+
+interface FormControlProps {
+  error?: string;
+  label: string;
+  children: React.ReactNode;
```

Comment:
Remove it from here.

and update component signature as `const FormControl: React.FC<PropsWithChildren<FormControlProps>>`